rinja_derive/src/input.rs

```diff
@@ -128,16 +129,17 @@ impl TemplateInput<'_> {
             let mut nested = vec![parsed.nodes()];
             while let Some(nodes) = nested.pop() {
                 for n in nodes {
-                    let mut add_to_check = |new_path: Rc<Path>| -> Result<(), CompileError> {
-                        if !map.contains_key(&new_path) {
+                    let mut add_to_check = |new_path| -> Result<(), CompileError> {
```

Comment:
Is it because the type changed or because the annotation is unnecessary? If it's the later, could you please keep the annotation? It makes it easier to answer the question "why so many `clone`?".

Reply:
No, the change wasn't needed. I first removed both the argument and return type, but I had to re-add the return type.

I re-added the argument type, too.